app/services/discovery_engine/evaluation/sample_query_set.rb

```diff
@@ -26,6 +26,30 @@ def create_empty_set
           )
       end
 
+      def import_from_bigquery(dataset_id:, table_id:)
```

Comment:
Minor: Perhaps `dataset_id` and `table_id` have default values? Unless we expect the names to be configurable in the rake task?

Reply:
I've added a couple of commits to try this.